File: src/main/java/ru/diasoft/micro/dofacade/mock/GetProductRes.java
package ru.diasoft.micro.dofacade.mock;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class GetProductRes extends InputParams {

    @JsonProperty("application")
    ProductApplicationRes productApplication;

}

File: src/main/java/ru/diasoft/micro/dofacade/json/product/GetProductRes.kt
package ru.diasoft.remote.json.product

import com.fasterxml.jackson.annotation.JsonProperty
import ru.diasoft.micro.dofacade.json.product.ProductApplicationRes
import ru.diasoft.micro.dofacade.json.InputParams

data class GetProductRes(

    @JsonProperty("application")
    val productApplication: ProductApplicationRes
) : InputParams()